frontend/gradle.properties

```diff
@@ -5,7 +5,7 @@ gson_version=2.8.5
 prometheus_version=0.9.0
 netty_version=4.1.53.Final
 slf4j_api_version=1.7.32
-slf4j_log4j_version=2.16.0
+slf4j_log4j_version=2.17.0
```

Comment:
@nskool Does it make sense to make this >= 2.17.0 so that future updates are taken care of?

Reply:
@chauhang I consulted with a senior engineer on this, and the recommendation is "I think we should use fixed version. We might accidentally pull in a bad or incompatible version during release." 

Another issue is - we don't have the release process automated. So although we may build the latest version on master branch, we'll still need to cut a release explicitly for now.